### DISCUSSION TITLE
Peter Gabriel I/O & The Court

### DISCUSSION THREAD

**Discussion by limnetik** [tin2tin/Pallaidium#104] [Show and tell]
I/O (Dark Side mix) by Peter Gabriel - DiffuseTogether Animation Challenge - I/O edition
https://youtu.be/abcYYRAyUVk

THE COURT (Bright Side mix) by Peter Gabriel - DiffuseTogether Animation Challenge - I/O edition
https://youtu.be/eG3cl-9kRH8

Both done before i got to know Pallaidium

**Comment by tin2tin** [comment]
Very nice, I also did a video for that competition (using Palladium(zeroscope)): https://www.youtube.com/watch?v=TvWMKLFAtGc

**Reply by limnetik** [reply]
I like the way you use imperfection to convey emotion, in a positive way. Beauty in the strangeness, good subject for ai tools. I did decide myself to use the weirdness of ai artifacts and not fight it in my videos; embrace the otherness of it all. Thanks
